CONTRIBUTING.md

```diff
@@ -0,0 +1,146 @@
+# Contributing to copilot-markdown-generator
+
+## Code Contributions
+
+### Requirements
+
+You will need to have installed on your machine:
+
+* PHP (v5.6.0 or above). If you're using macOS, you already have this installed.
+* Composer. [Get Composer][Composer]
```

Comment:
Maybe mention that `composer` can be brew installed? Just a suggestion.

Reply:
@nigelhanlon Yes, I can't find anywhere in the Composer docs where it says that there is official Homebrew support, but it looks up to date.